blueprints/sara2.py

```diff
@@ -20,16 +20,16 @@ def posts_user(user_id):
     response= requests.get(f"https://jsonplaceholder.typicode.com/users/{user_id}/posts")
     
     if user_id is None:
-        return jsonify(error= "user_id is None")
+        return jsonify(error= "user_id is null")
     
     if int(response.status_code)//100 != 2:
         return jsonify(error= "Invalid request")
     
     return response.json()
 
-@sara2.route('/user/<user_id>/comments', methods=["GET"])
-def comments_user(user_id):
-    response= requests.get(f"https://jsonplaceholder.typicode.com/posts/{user_id}/comments")
+@sara2.route('/user/<post_id>/comments', methods=["GET"])
```

Comment:
The `user` has nothing to do with `post_id` therefore the route should be changed entirely